sources/api/views/UserView.ts

```diff
@@ -16,6 +16,13 @@ export class UserView {
         };
     }
 
+    public static formatPendingUser(user: PendingUser): any {
+        return {
```

Comment:
Maybe create a `PendingUserView` instead of using `UserView` ?

Don't know what is the best 😕